aea/decision_maker/messages/transaction.py

```diff
@@ -279,7 +279,7 @@ def check_consistency(self) -> bool:
                 self.Performative.REJECTED_SETTLEMENT,
                 self.Performative.FAILED_SETTLEMENT,
             }:
-                assert len(self.body) == 11
+                assert len(self.body) == 11 or len(self.body) == 12
```

Comment:
No we should not change that. Please add the `tx_nonce` to the `info` dictionary.

Reply:
That way the `TransactionMessage` remains unchanged and you can do `tx_message.info.get("tx_nonce")`